### PR TITLE
Prototype of upcoming movies application

### PR DESCRIPTION
**Pull request by miszu**
Created a prototype in Xamarin Forms for an app showing upcoming movies from TMDB. Users are able to see the movies (their name, description, images, genres, release date) and find movies by title. The app is working on both iOS and Android.

### CODE REVIEW

File: Movies/Movies/Converters/StringsArrayToCSVConverter.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Movies.Converters
+{
+    public class StringsArrayToCSVConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            return String.Join(", ", value as string[]);
```
Comment (arctouch-victorbotamedi) [inline Movies/Movies/Converters/StringsArrayToCSVConverter.cs:10]: Really nice solution doing it as an `IValueConverter`

File: Movies/Movies/DTO/MovieDTO.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        [DataMember(Name = "id")]
+        public int Id { get; set; }
+
+        [DataMember(Name = "title")]
+        public string Title { get; set; }
+
+        [DataMember(Name = "popularity")]
+        public double Popularity { get; set; }
+
+        [DataMember(Name = "poster_path")]
```
Comment (arctouch-victorbotamedi) [inline Movies/Movies/DTO/MovieDTO.cs:18]: You can use a `SnakeCaseNamingStrategy` instead of declaring a `DataMember` for each property.

**Review by arctouch-victorbotamedi — COMMENTED** [review]
Nice solution overall, easy to understand and well written. Great job doing the error handling.

There is only a small layout problem in Android devices where the white navbar is hiding the white search icon.

Well done!

**Comment by miszu** [comment]
Thanks @arctouch-victorbotamedi :)

**Pull request merged**